R/sim_gs_n.R

```diff
@@ -38,6 +38,15 @@
 #'   positional argument to each test function provided.
 #' @param cut A list of cutting functions created by [create_cut()], see
 #'   examples.
+#' @param original_design A design object from the gsDesign2 package, which is required when users
+#' want to calculate updated bounds. The default is NULL leaving the updated bounds uncalculated.
+#' @param ia_alpha_spending Spend alpha at interim analysis based on
+#' - `"min_planned_actual"`: the minimal of planned and actual alpha spending.
+#' - `"actual"`: the actual alpha spending.
+#' @param fa_alpha_spending If targeted final event count is not achieved (under-running at final analysis),
+#' specify how to do final spending. Generally, this should be specified in analysis plan.
+#' - `"info_frac"` = spend final alpha according to final information fraction
+#' - `"full_alpha"` = spend full alpha at final analysis.
```

Comment:
Minor nit: the formatting for the other arguments indents each line with two spaces